amagaki.js

```diff
@@ -18,6 +18,10 @@ module.exports = function (pod) {
         path: '/static/css/',
         staticDir: '/dist/css/',
       },
+      {
+        path: '/static/',
```

Comment:
I think we should combine this with the previous line? So I believe you can delete the lines 17-20 in favor of this updated rule.